### ISSUE TITLE
Improving Generalization of Synthetically Trained Sonar Image Descriptors for Underwater Place Recognition. (arXiv:2308.01058v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#27113]
Improving Generalization of Synthetically Trained Sonar Image Descriptors for Underwater Place Recognition. (arXiv:2308.01058v1 [cs.CV])<br>
https://ift.tt/7QenUKm<br>
<p>Autonomous navigation in underwater environments presents challenges due to
factors such as light absorption and water turbidity, limiting the
effectiveness of optical sensors. Sonar systems are commonly used for
perception in underwater operations as they are unaffected by these
limitations. Traditional computer vision algorithms are less effective when
applied to sonar-generated acoustic images, while convolutional neural networks
(CNNs) typically require large amounts of labeled training data that are often
unavailable or difficult to acquire. To this end, we propose a novel compact
deep sonar descriptor pipeline that can generalize to real scenarios while
being trained exclusively on synthetic data. Our architecture is based on a
ResNet18 back-end and a properly parameterized random Gaussian projection
layer, whereas input sonar data is enhanced with standard ad-hoc
normalization/prefiltering techniques. A customized synthetic data generation
procedure is also presented. The proposed method has been evaluated extensively
using both synthetic and publicly available real data, demonstrating its
effectiveness compared to state-of-the-art methods.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
